cancat/__init__.py

```diff
@@ -641,7 +641,7 @@ def CANreplay(self, start_bkmk=None, stop_bkmk=None, start_msg=0, stop_msg=None,
             delta_correction = newstamp - laststamp
 
             if timing == TIMING_INTERACTIVE:
-                raw_input("%s\nPress Enter to Transmit" % reprCanMsg(idx, ts, arbid, data))
+                raw_input("Press Enter to Transmit next message")
```

Comment:
i prefer showing what is about to be sent.  not what was just sent.  if you have a better way to communicate that, great. but please don't show what was just sent.  or have a way to choose which way to go.
i'm ok with reversing the order "Press Enter to Transmit:  %s".